src/app/sharedUI/items/ArtworkGridItem.tests.tsx

```diff
@@ -0,0 +1,100 @@
+import { renderWithWrappers } from "shared/tests/renderWithWrappers"
+import { ArtworkGridItem } from "./ArtworkGridItem"
+
+describe("ArtworkGridItem", () => {
+  const TestRenderer = () => {
+    const data = useLazyLoadQuery<ArtworkGridItem_artwork_TestQuery>(
+      graphql`
+        query ArtworkGridItem_artwork_TestQuery @relay_test_operation {
+          artwork(id: "some-id") {
+            ...ArtworkGridItem_artwork
+          }
+        }
+      `,
+      {}
+    )
+
+    return <ArtworkGridItem artwork={data.artwork!} />
+  }
+
+  it("should NOT HIDE the availabilty status dot if the 'Presantation Mode = OFF' and 'Hide Works Availability' switch = OFF", async () => {
+    const { container } = renderWithWrappers(<TestRenderer />)
+
+    __globalStoreTestUtils__?.injectState({
+      presentationMode: {
+        isPresentationModeEnabled: false,
+        isHideWorksAvailabilityEnabled: false,
+      },
+    })
+
+    await mockEnvironmentPayloadMaybe({
+      Artwork: () => ({
+        availability: "for sale",
+      }),
+    })
+
+    expect(container.findByType(Text).props.children[0].props).toBeTruthy()
+  })
+
+  it("should NOT HIDE the availabilty status dot if the 'Presantation Mode = OFF' and 'Hide Works Availability' switch = ON", async () => {
+    const { container } = renderWithWrappers(<TestRenderer />)
+
+    __globalStoreTestUtils__?.injectState({
+      presentationMode: {
+        isPresentationModeEnabled: false,
+        isHideWorksAvailabilityEnabled: true,
+      },
+    })
+
+    await mockEnvironmentPayloadMaybe({
+      Artwork: () => ({
+        availability: "for sale",
+      }),
+    })
+
+    expect(container.findByType(Text).props.children[0].props).toBeTruthy()
+  })
+
+  it("should NOT HIDE the availabilty status dot if the 'Presantation Mode = ON' and 'Hide Works Availability' switch = OFF", async () => {
+    const { container } = renderWithWrappers(<TestRenderer />)
+
+    __globalStoreTestUtils__?.injectState({
+      presentationMode: {
+        isPresentationModeEnabled: true,
+        isHideWorksAvailabilityEnabled: false,
+      },
+    })
+
+    await mockEnvironmentPayloadMaybe({
+      Artwork: () => ({
+        availability: "for sale",
+      }),
+    })
+
+    expect(container.findByType(Text).props.children[0].props).toBeTruthy()
+  })
+
+  it("should HIDE the availabilty status dot if the 'Presantation Mode = ON' and 'Hide Works Availability' switch = ON", async () => {
+    const { container } = renderWithWrappers(<TestRenderer />)
+
+    __globalStoreTestUtils__?.injectState({
+      presentationMode: {
+        isPresentationModeEnabled: true,
+        isHideWorksAvailabilityEnabled: true,
+      },
+    })
+
+    await mockEnvironmentPayloadMaybe({
+      Artwork: () => ({
+        availability: "for sale",
+      }),
+    })
+
+    expect(container.findByType(Text).props.children[0].props).toBeUndefined()
```

Comment:
```suggestion
    expect(container.findByType(Text).props.children[0].props).toBeFalsy()
```

lets do this so its consistent with above?